packages/ui/src/lib/components/persona.svelte

```diff
@@ -66,25 +58,35 @@
 		display: flex;
 		flex-direction: column;
 		gap: var(--spacing-8);
+		min-height: 92px;
+
+		@media (min-width: 398px) {
```

Comment:
Good enough for now, but I would prefer to use `@container` media query here. So when the container is `350px` or wider, it should have different font sizes. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries